styleguide.config.js

```diff
@@ -0,0 +1,22 @@
+const {createConfig, babel, postcss} = require("webpack-blocks");
+
+module.exports = {
+    webpackConfig: createConfig([babel(), postcss()]),
```

Comment:
Interesting! Hadn't seen this before. Are we using postcss? I'm not sure I see it anywhere?

Reply:
I just copy pasted from the styleguidist quick start.